js2_1/js_2_1.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h1>FizzBuzz問題</h1>
+
+  <div>
+    <form>
+      <div>
+      <label for="name">FizzNum:</label>
+      <input type="text" id="Fizz" placeholder="整数値を入力してください">
+      </div>
+
+      <div>
+      <label for="name">BuzzNum:</label>
```

Comment:
対応する HTML Element の Id を指定する必要があるので、以下のようになっている必要があります。
```suggestion
      <label for="Buzz">FizzNum:</label>
```